.github/workflows/automated_release.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - develop
       - master
+      - dpl-753-2-process-qc-fields
```

Comment:
I guess this can be removed. It might be a good idea to add the asset_dispatch.yml as per other repos so you can create a release on the fly without having to change the automated_release action

Reply:
Done